recipes/flake8-quotes/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "flake8-quotes" %}
+{% set version = "0.8.1" %}
+{% set sha256 = "668ec2fb0fbf1574a95f49e393364f8a114c7180e5cedc7377c5f4b5257e00fb" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  preserve_egg_dir: True
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - flake8 >=3.0.4,<4.0.0
```

Comment:
Can we please drop `flake8` from `build`? Should only be needed during the install.